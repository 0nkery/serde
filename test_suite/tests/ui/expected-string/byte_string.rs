#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
struct S {
    #[serde(rename = b"byte string")]
    byte_string: (),
}

fn main() {}
