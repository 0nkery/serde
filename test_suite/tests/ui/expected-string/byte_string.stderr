error: expected serde rename attribute to be a string: `rename = "..."`
 --> $DIR/byte_string.rs:6:22
  |
6 |     #[serde(rename = b"byte string")]
  |                      ^^^^^^^^^^^^^^

error: aborting due to previous error

